js/plates-model/field.js

```diff
@@ -241,12 +241,13 @@ export default class Field extends FieldBase {
       return
     }
 
-    if (this.isOcean && this.density < field.density) {
+    // Higher density plates subduct
+    if (this.isOcean && this.density > field.density) {
```

Comment:
This was one big change I made, which seems to have broken the serialization tests. Previously, it seems like lower density plates were subducting instead of higher density plates, which I believe is backwards. These changes, along with a small rendering change, should fix that

Reply:
btw, presets need to be updated too (all the densities need to be inverted):
https://github.com/concord-consortium/plate-tectonics-3d/blob/master/js/presets.js
so they work as they used to.

Reply:
Re failing tests, I'd try to update densities in test case too. It shouldn't fail in general. But if it won't fail after densities update, it means that this problem is probably not related to this PR. Also, a test should fail when you invert densities in the master branch. 😉